src/richie/apps/courses/templates/courses/cms/fragment_course_runs_list.html

```diff
@@ -9,11 +9,8 @@
                 {% if course != run.direct_course %}
                     <a href="{{ run.direct_course.extended_object.get_absolute_url }}">
                 {% endif %}
-                {% if run.start  %}
-                    {% blocktrans with title=run.title|capfirst start=run.start|date:'DATE_FORMAT'|default:'...' end=run.end|date:'DATE_FORMAT'|default:'...' %}{{ title }} from {{ start }} to {{ end }}{% endblocktrans %}
-                {% else %}
-                    {{ run.title|capfirst }}
-                {% endif %}
+                {% blocktrans with title=run.title|default:"" start=run.start|date:'DATE_FORMAT'|default:'...' end=run.end|date:'DATE_FORMAT'|default:'...' asvar run_title %}{{ title }} from {{ start }} to {{ end }}{% endblocktrans %}
```

Comment:
Because now there might be not title AND no start date :man_shrugging: 
Also a course run with no start date is not shown on the public page and only visible by editors... so I find it clearer to show that some information is missing. If I display only the title like before, it doesn't even look like a course run!

Reply:
Oh ok! So yes it is clearly better to display `From ... to ...` to the editor.